torch/optim/adamw.py

```diff
@@ -479,7 +481,9 @@ def _multi_tensor_adamw(
     if len(params) == 0:
         return
 
-    if capturable:
+    # If we are compiling, the compiler will determine if cudagraphs
+    # is applicable
```

Comment:
Now all of these could be replaced
```suggestion
    # If compiling, the compiler will handle cudagraph checks, see note [torch.compile x capturable]
```